tests/test_coordinator.py

```
@@ -83,6 +83,22 @@ async def test_connect_success(
         assert result is True
         assert coordinator.controller is not None
 
+    async def test_connect_closes_stale_connections_before_connect(
+        self,
+        hass: HomeAssistant,
+        mock_config_entry,
+        mock_coordinator_connected,
```

Ruff (0.15.1)

[warning] Unused method argument: `mock_coordinator_connected`

(ARG002)